src/index.ts

```diff
@@ -0,0 +1,16 @@
+import { readFileSync } from 'node:fs'
+import { fetchMyIp } from './fetchMyIp.js'
+import { getAllData, storeIp } from './ipStore.js'
+import { ipToCountryCode } from './ipToCountryCode.js'
+import { printDataWithCountryCodes } from './printDataWithCountryCodes.js'
+
+const configFile = process.env.CONFIG_FILE || 'config.json'
+const configString = readFileSync(configFile).toString()
+const config = JSON.parse(configString)
+
+const userName = process.argv[3]
+const userIP = await fetchMyIp(config.ifconfigUrl)
```

Comment:
defined as `ifconfigUri` in the config, so it will not be found
we need better typing to detect these errors